zebedee-cms/src/test/java/com/github/onsdigital/zebedee/api/StubServletOutputStream.java

```diff
@@ -0,0 +1,35 @@
+package com.github.onsdigital.zebedee.api;
```

Comment:
Maybe move it up to a different more generic package rather than make it specific to apis?

Reply:
it's only used by api tests

Reply:
I know, but we do not place classes in packages where they are used (at the moment) but where they belong? This is not api specific and could be used from anywhere else?

Reply:
what package do you suggest?

Reply:
I can't actually see a more suitable place, it's next to where it's used and unlikely to be used other than for API testing.